src/Deployer.php

```diff
@@ -77,7 +77,12 @@ public function addTarget(string $name, string $path, string $publicURL = null,
         $host = host($name)->setDeployPath($path);
 
         if (null !== $this->hostname) {
-            $host->setHostname($this->hostname);
+            [$hostname, $port] = explode(':', $this->hostname, 2) + [null, null];
```

Comment:
@aschempp `$this->hostname` cannot be `null` here that's why I didn't do it.

Reply:
It would work just fine: https://3v4l.org/NveI2

Reply:
> Warning: Undefined array key 1 in /in/NveI2 on line 3
> string(8) "hostname"
> NULL

The warning is exactly what my changes would prevent!

Reply:
Yes, can we merge finally or how long does this PR need to be open?